### ISSUE TITLE
abc-quickstart  -  mismatch of roster file names i n  config  vs what's c reated

### ISSUE THREAD

**Issue by lwasser** [earthlab/abc-classroom#383]
The file provided is `sample_roster.csv`
the file in the config  is  `classroom_roster.csv`

we probably should make sure both  are sample_roster? or classroom_roster?

<img width="520" alt="Screen Shot 2021-02-18 at 6 35 35 PM" src="https://user-images.githubusercontent.com/7649194/108444620-37730b00-7218-11eb-8ddb-4fde5c0d8fa7.png">
<img width="747" alt="Screen Shot 2021-02-18 at 6 35 43 PM" src="https://user-images.githubusercontent.com/7649194/108444613-35a94780-7218-11eb-8075-7778b2774c1c.png">

**Comment by kcranston** [comment]
Yes, the naming should be consistent throughout, but I wonder if the name should be abc specific, e.g. `abc_roster.csv`.  When you download the GitHub Classroom roster, the default name of that file is `classroom_roster.csv`. When I first set this up, I thought we could just use that file, so I used the same filename for the `abc-classroom` default. 

But, if we are going to write a roster creation utility that converts the github roster to an abc roster, having them both with the same default name seems confusing. 

* download roster from GitHub Classroom = `classroom_roster.csv`
* run rabc oster creation script = new file `abc_roster.csv`
